src/OrbitGl/ThreadTrack.cpp

```diff
@@ -69,22 +68,27 @@ std::string ThreadTrack::GetThreadNameFromTid(uint32_t thread_id) {
   ;
 }
 
-void ThreadTrack::InitializeNameAndLabel() {
+const std::string_view kAllThreads = " (all_threads)";
+const std::string ThreadTrack::GetLabel() const {
+  auto thread_id = GetThreadId();
```

Comment:
This variable can be moved down when it's first used. You can also consider to eliminiate the variable. It's only used once.

Reply:
It's used 3 times, you might miss the ones inside the ifs

Reply:
Ah yeah. Nothing to see here. :see_no_evil: